### PR TITLE
Feat: more extension details

### PR DESCRIPTION
**Pull request by arcbtc**
For new users its disorientating to install extensions and not know what they do. This pr makes use of an extended `config.json` in extensions https://github.com/lnbits/lnurldevice/blob/main/config.json
The pr will build a `More` dialogue that can display images, videos, extended description. When we have reviews this will also be how they are accessed.

You can test with https://raw.githubusercontent.com/lnbits/tpos/main/manifest.json

This pr will also enable a seperate splash page to be built like https://en-gb.wordpress.org/plugins/

### CODE REVIEW

File: lnbits/core/views/extension_api.py [hunk 1]
```diff
@@ -189,6 +190,24 @@ async def get_extension_releases(ext_id: str):
         ) from exc
 
 
+@extension_router.get("/{ext_id}/details")
```
Comment (dni) [inline lnbits/core/views/extension_api.py:193]: this can be done on the frontend only

**Comment by motorina0** [comment]
Is there a manifest.json that I can use for testing?

File: lnbits/core/views/generic.py [hunk 2]
```diff
@@ -136,16 +135,22 @@ async def extensions_install(
...
         extensions = [
             {
                 "id": ext.id,
                 "name": ext.name,
                 "icon": ext.icon,
                 "shortDescription": ext.short_description,
-                "extendedDescription": ext.description_md or None,
-                "immages": ext.images or None,
-                "contributors": ext.contributors or None,
+                "immages": ext.images if ext.images else None,
```
Comment (dni) [inline lnbits/core/views/generic.py]:
```suggestion
                "images": ext.images if ext.images else None,
```

Comment (dni) [reply lnbits/core/views/generic.py]: isnt ext.images `None` this if makes no sense, imho?

File: lnbits/core/templates/core/extensions.html [hunk 3]
```diff
@@ -1089,6 +1085,7 @@ <h6 class="q-my-sm" v-text="extensionConfig.short_description"></h6>
         ...e,
         inProgress: false
       }))
+      console.log(this.extensions)
```
Comment (dni) [inline lnbits/core/templates/core/extensions.html]: log

**Pull request closed without merging**